packages/core/system/src/types.ts

```
@@ -2,3 +2,381 @@
+  readonly offset: number;
+  constructor(
+    year: number,
+    month: number,
+    day: number,
+    timeZone: string,
+    offset: number,
+    hour?: number,
+    minute?: number,
+    second?: number,
+    millisecond?: number,
+  );
```

Biome (1.9.4)

[error] Don't use the constructor method in interfaces.

Interfaces define a contract, not an implementation. Thus, including a constructoris not appropriate.

(lint/suspicious/noMisleadingInstantiator)